PicApp_git/PicApp/routes/web.php

```diff
@@ -20,4 +20,10 @@
     Route::post('/top', 'PostsController@store');
 
     Route::get('/posts', 'PostsController@showCreateForm')->name('post.create');
+
+    Route::get('/posts/{post_id}', 'PostsController@showPostDetail')->name('detail');
+
+    Route::post('/posts/{post_id}', 'PostsController@storeComment');
```

Comment:
ぱっとみ投稿のルーティングに見えるので、下記のほうが良いかもです
```suggestion
    Route::post('/posts/{post_id}/comments', 'PostsController@storeComment');
```